projects/canopy/src/lib/skeleton/skeleton.directive.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Directive,
+  ElementRef,
+  Renderer2,
+  Input,
+  HostBinding,
+  OnDestroy,
+} from '@angular/core';
+
+@Directive({
+  selector: '[lgSkeleton]',
+})
+export class LgSkeletonDirective implements OnDestroy {
+  private changes: MutationObserver;
+
+  @Input() lgSkeletonWidth: string = null;
+  @Input() lgSkeletonHeight: string = null;
+  @Input() lgSkeletonRightAlign = false;
+
+  @HostBinding('class.lg-skeleton') skeletonClass = true;
+  @HostBinding('class.lg-skeleton--right')
+  get alignClass() {
+    return this.lgSkeletonRightAlign;
```

Comment:
Not sure if this adds complexity but you might be able to assign this automatically.
```ts
  @HostBinding('class.lg-skeleton--right')
  get alignClass() {
    return window
      .getComputedStyle(this.hostElement.nativeElement)
      .getPropertyValue('text-align') === 'right';
```

Reply:
Yeah I did think about it... but its not just `text-align`, could be a flex item that is right-aligned with margin etc. So this gives the consumer the flexibility to use any layout mechanism and still get the alignment to the right, if needed.